### PR TITLE
Add python3-transformers-pip

### PR DESCRIPTION
**Pull request by greinet**
Please add the following dependency to the rosdep database.

## Package name:

transformers

## Package Upstream Source:

https://pypi.org/project/transformers/

## Purpose of using this:

Transformers provides thousands of models for various machine-learning tasks like text, image or audio. It is backed by Jax, PyTorch and Tensorflow and provides seamless integration.

Distro packaging links:

## Links to Distribution Packages

- Debian: https://pypi.org/project/transformers/
- Ubuntu: https://pypi.org/project/transformers/

### CODE REVIEW

File: rosdep/python.yaml [hunk 1]
```diff
@@ -5321,6 +5321,13 @@ python-tqdm:
     xenial:
       pip:
         packages: [tqdm]
+python3-transformers-pip:
```
Comment (clalancette) [inline rosdep/python.yaml:5324]: This is failing CI because this needs to be after the `python3-transforms3d-pip` key.

**Review by clalancette — CHANGES_REQUESTED** [review]

File: rosdep/python.yaml (hunk 1 above)
Comment (greinet) [reply rosdep/python.yaml]: Oh sorry, fixed it.

**Review by clalancette — CHANGES_REQUESTED** [review]
Ah sorry, I got this wrong.  The key is in the complete wrong part of the file; because of the `python3` prefix, it needs to be among the `python3` keys, not the `python` keys.  If you move it, that should hopefully fix it.

**Comment by greinet** [comment]
I added the 3 to the prefix after creating the PR and didn't think about it, now it should be in the right spot I hope.

**Review by clalancette — APPROVED** [review]

**Review by methylDragon — APPROVED** [review]
Exciting!

**Pull request merged**